### ISSUE TITLE
[Quetion about the Dinoloss] Is this the correct way to implement original DINO LOSS??

### ISSUE THREAD

**Issue by Backdrop9019** [alinlab/SelfPatch#8]
From how i knew, In Dino loss Teacher Model uses only global views. and Student Model uses both global and local views.
However in your code. Teacher Model uses both global and local views like student model.

Did you intentionally add local views in teacher model? 
If so how is this same as original Dino loss?

Input code :
`teacher_output = [teacher(torch.cat(images[:2]), head_only=True, loc=True), teacher(torch.cat(images[2:]), head_only=True, loc=True)]`
loss code :
`teacher_cls = teacher_output[0][0].chunk(2) + teacher_output[1][0].chunk(self.ncrops-2)`
